packages/arb-token-bridge-ui/src/pages/api/screenings.ts

```diff
@@ -36,19 +36,15 @@ function isBlocked(response: ExternalApiResponse): boolean {
     result => result.addressRiskIndicators.length > 0
   )
 
-  if (riskIdentified) {
-    let isSevere = false
-    response.forEach(chainInfo => {
-      chainInfo.addressRiskIndicators.forEach(riskIndicator => {
-        if (riskIndicator.categoryRiskScoreLevel > 1) isSevere = true
-      })
-    })
-
-    return isSevere
-  } else {
-    // no risk identified
+  if (!riskIdentified) {
     return false
   }
+
+  return response.some(chainInfo => {
+    return chainInfo.addressRiskIndicators.some(
+      riskIndicator => riskIndicator.categoryRiskScoreLevel > 1
```

Comment:
should it be >= 1 or >0 instead of >1?

Reply:
I think 1 is a small risk and we want to block only addresses higher than that?

Reply:
Yes, but we need to revisit this in the docs properly.